frontend/plugins/frontline_ui/src/modules/integrations/call/components/CallIntegrationEdit.tsx

```
@@ -22,34 +24,77 @@ export const CallIntegrationSheetEdit = () => {
-        channelId: callsIntegrationDetail.channelId,
+        name: integrationDetail.name || '',
+        phone: callsIntegrationDetail?.phone || '',
+        websocketServer: callsIntegrationDetail?.wsServer || '',
+        queues: callsIntegrationDetail?.queues || '',
+        operators: callsIntegrationDetail?.operators || '',
+        brandId: integrationDetail.brandId,
+        channelIds:
+          integrationDetail.channels?.map((channel) => channel._id) || [],
       });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)